dht-node/src/config/index.ts

```diff
@@ -9,7 +9,7 @@ const constants = {
   LOG_LEVEL: process.env.LOG_LEVEL || 'info',
   CONFIG_VERSION: {
     DEFAULT: 'DEFAULT',
-    EXPECTED: 'v2.2023-02-07',
+    EXPECTED: 'v3.2023-04-26',
```

Comment:
a change here must be refected in `deployment/bare_metal/.env.dist`

Reply:
done